content/en/docs/setup/independent/high-availability.md

```diff
@@ -446,40 +452,41 @@ Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**).
 
 ## Run kubeadm init on master0 {#kubeadm-init-master0}
 
-1.  In order for kubeadm to run, you first need to write a configuration file:
-      ```none
-      cat >config.yaml <<EOF
-      apiVersion: kubeadm.k8s.io/v1alpha1
-      kind: MasterConfiguration
-      api:
-      advertiseAddress: <private-ip>
-      etcd:
-      endpoints:
-      - https://<etcd0-ip-address>:2379
-      - https://<etcd1-ip-address>:2379
-      - https://<etcd2-ip-address>:2379
-      caFile: /etc/kubernetes/pki/etcd/ca.pem
-      certFile: /etc/kubernetes/pki/etcd/client.pem
-      keyFile: /etc/kubernetes/pki/etcd/client-key.pem
-      networking:
-      podSubnet: <podCIDR>
-      apiServerCertSANs:
-      - <load-balancer-ip>
-      apiServerExtraArgs:
-      apiserver-count: "3"
-      EOF
-      ```
+In order for kubeadm to run, you first need to write a configuration file:
```

Comment:
Anyone opposed to just removing the numbering from this section altogether? Or alternative going with something like a bolded **Step 1**, etc as @tfogo suggested [here](https://github.com/kubernetes/website/pull/8663#issuecomment-390819141). It's a relative brief section anyway (two "steps"), so IMO the numbering doesn't add that much in the way of clarity; but the current workaround does cause problems when copy pasting (not to mention docs code maintenance).

No strong opinions on this one; just thought it would help simplify things.